jsparagus/actions.py

```diff
@@ -6,6 +6,8 @@
 from .grammar import Element, InitNt, Nt
 from . import types, grammar
 
+StateId = int
```

Comment:
(nit) Please change aps.py and parse_table.py to import `StateId` from here so there aren't two definitions.

Reply:
Is there a better location where these types could live in? I find it strange that the StateId lives in the action.py file.